app/models/solidus_bolt/gateway.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module SolidusBolt
+  class Gateway
+    def initialize(options = {}); end
+
+    def authorize(_amount, _payment_source, _gateway_options)
+      raise NotImplementedError, 'authorize method has not been implemented in SolidusBolt::Gateway class'
+    end
+
+    def capture(_float_amount, _order_number, _gateway_options)
```

Comment:
based on a review I got from Daniele on Acima, it should be `_response_code` instead of `_order_number`